src/components/displayAllComponent.js

```diff
@@ -37,7 +37,7 @@ class DisplayAllComponent extends Component {
           name: setting.name,
           optimisedSvg: await optimizeSvg(
             setting.dataurl,
-            this.props.svgObject.plugins
+            null
```

Comment:
is this supposed to be null?

Reply:
Yes, if it null it will apply all the optimisation options.

Reply:
cool

Reply:
So would you agree that these  15 lines of code look like an exact copy of one another except one line? Do you think there could be a way to make this less duplicated

Reply:
surely there is, but I see it more readable this way even to someone with less knowledge of js.

Reply:
It have been shortened and pushed.